src/module_type_translator.ts

```diff
@@ -410,7 +410,15 @@ export class ModuleTypeTranslator {
   getFunctionTypeJSDoc(fnDecls: ts.SignatureDeclaration[], extraTags: jsdoc.Tag[] = []):
       [jsdoc.Tag[], string[]] {
     const typeChecker = this.typeChecker;
-    const newDoc = extraTags;
+
+    // De-duplicate tags and docs found for the fnDecls.
```

Comment:
Will this deduplicate @params, which we do want multiple of?  (E..g if the function has @param x and @param y)

Reply:
No, tags called `@param` or `@return` are special cased below. We also have plenty of tests for this, I know because I broke them at first :-)